plugins/orchestrator/src/components/Paragraph/Paragraph.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { PropsWithChildren } from 'react';
+
+import { Typography } from '@material-ui/core';
+import { Variant } from '@material-ui/core/styles/createTypography';
+
+export const Paragraph = (
+  props: PropsWithChildren<{ variant?: Variant | 'inherit' }>,
+) => {
+  return (
+    <Typography
+      style={{ marginTop: '14px', marginBottom: '14px' }}
```

Comment:
useStyle()

Reply:
There is a refactoring going on and this component will likely be replaced with new code.